docs/docs/integrating-with-other-libraries.md

```diff
@@ -0,0 +1,205 @@
+  }
+
+  componentWillUnmount() {
+    // cleanup
+  }
+
+  render() {
+    return <div ref={el => this.$el = $(el)} />
+  }
+}
+```
+A [ref](https://facebook.github.io/react/docs/refs-and-the-dom.html) is used to pass the underlying DOM element to the plugin. The `<div>` element has no properties or children, so React has no reason to update it.
+
+The component still has to be unmounted, which provides one final opportunity for conflict. If the plugin does not provide a method for cleanup, you will probably have to provide your own, remembering to remove any event listeners the plugin registered to prevent memory leaks.
+
+To demonstrate these concepts let's write a minimal wrapper for the plugin [Chosen](https://github.com/harvesthq/chosen), which augments `<select>` inputs.
+
+Chosen does not render the initial select, so it is up to React to do so. What it does do is hide actual select and create its own control, notifying the original of changes using jQuery. Because it is Chosen maintaining the state, it is easiest to implement the wrapper using an [uncontrolled component.](https://facebook.github.io/react/docs/uncontrolled-components.html)
+
+```js
+class Chosen extends React.Component {
+  handleUpdate = event => this.props.update(event.target.value);
+
+  componentDidMount() {
+    this.$el.chosen();
+    this.$el.on("change", this.handleUpdate);
+  }
+
+  componentDidUpdate() {
+    this.$el.trigger("chosen:updated");
+  }
+
+  componentWillUnmount() {
+    this.$el.off("change", this.handleUpdate);
+    this.$el.chosen("destroy");
+  }
+
+  render() {
+    return (
+      <select ref={el => this.$el = $(el)}>
+        {this.props.children}
+      </select>
+    );
+  }
+}
+```
+
+1. Change listeners need to be setup inside `componentDidMount()` and torn down in `componentWillUnmount` as the events are emitted by the plugin use jQuery, which is outside the React event system.
+
+2. Chosen needs to be notified of any changes to the original select or its children. This is done in `componentWillUpdate()` by triggering a `"chosen:updated"` event.
+
+3. When the component is unmounted, use the cleanup method provided by Chosen. This removes the custom select control and restores the actual `<select>` as well as removing any event listeners the plugin registered.
+
+[Try it on CodePen.](http://codepen.io/wacii/pen/ygzxjG?editors=0010)
+
+## Backbone
+
+### Embedding React in a Backbone View
+
+Creating a backbone wrapper around a react component is easy thanks to the flexibility of `ReactDOM.render()`. While a typical application usually calls the `render` method just once, it may be called repeatedly to update props or to create multiple component trees.
+
+Each view will have an associated component, and when the view renders, `ReactDOM.render()` is used to render the component into the view's `el`.
+
+```js
+function Component({ text }) {
+  return <p>{text}</p>;
+}
+
+const ComponentView = Backbone.View.extend({
+  render() {
```

Comment:
I added a section about rendering with strings/templates so the `render` function doesn't come out of nowhere. Not sure if that's enough.